### ISSUE TITLE
Missing dependency definition to Alien::Sodium

### ISSUE THREAD

**Issue by jpawlowski** [ajgb/crypt-nacl-sodium#8]
Would you mind to add Alien::Sodium to the actual dependency list in the meta data so that it will automatically be installed by cpan before Crypt::NaCl::Sodium?

**Comment by plicease** [comment]
Recommended prereq will not consistently work depending on the cpan client.  ABMB needs to be added as a configure time prereq to `Alien::Sodium`.  `Crypt::NACL::Sodium` shouldn't care if `Alien::Sodium` uses ABMB or `Alien::Build`, or rolls its own install procedure.  This PR
https://github.com/ajgb/alien-sodium/pull/4/files
and a release will accomplish that.

**Comment by plicease** [comment]
`Alien::Sodium` is already a configure prereq, which is he correct phase.  Also `Alien::Sodium` has a new maintainer and the ABMB issue has been resolved in the most recent release.  I believe this issue can be closed.

**Comment by tonvoon** [comment]
As `Alien::Sodium` is referenced here, I think the update to 2.0.0 has broken the build of Crpyt::NaCl::Sodium. Error is:

`
cp lib/Crypt/NaCl/Sodium/aead.pod blib/lib/Crypt/NaCl/Sodium/aead.pod
Running Mkbootstrap for Sodium ()
chmod 644 "Sodium.bs"
"/usr/bin/perl" -MExtUtils::Command::MM -e 'cp_nonempty' -- Sodium.bs blib/arch/auto/Crypt/NaCl/Sodium/Sodium.bs 644
"/usr/bin/perl" "/tmp/ton123/opt/opsview/perl/lib/perl5/ExtUtils/xsubpp"  -typemap '/usr/share/perl/5.22/ExtUtils/typemap' -typemap '/home/vagrant/projects/opsview-perl-external/cpanm_work/work/1597354719.22271/Crypt-NaCl-Sodium-1.0.8.0/typemap'  Sodium.xs > Sodium.xsc
mv Sodium.xsc Sodium.c
x86_64-linux-gnu-gcc -c  -I. -I/tmp/ton123/opt/opsview/perl/lib/perl5/x86_64-linux-gnu-thread-multi/auto/share/dist/Alien-Sodium/include  -D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fwrapv -fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -O2 -g   -DVERSION=\"1.0.8.0\" -DXS_VERSION=\"1.0.8.0\" -fPIC "-I/usr/lib/x86_64-linux-gnu/perl/5.22/CORE"  -DAES256GCM_IS_AVAILABLE Sodium.c
Sodium.xs: In function ‘XS_Crypt__NaCl__Sodium__stream_AES128CTR_NONCEBYTES’:
Sodium.xs:4538:18: error: ‘crypto_stream_aes128ctr_NONCEBYTES’ undeclared (first use in this function)
         RETVAL = crypto_stream_aes128ctr_NONCEBYTES;
                  ^
Sodium.xs:4538:18: note: each undeclared identifier is reported only once for each function it appears in
Sodium.xs: In function ‘XS_Crypt__NaCl__Sodium__stream_AES128CTR_KEYBYTES’:
Sodium.xs:4545:18: error: ‘crypto_stream_aes128ctr_KEYBYTES’ undeclared (first use in this function)
         RETVAL = crypto_stream_aes128ctr_KEYBYTES;
                  ^
`

I'm afraid I don't understand the error very much.

**Comment by genio** [comment]
Yes, the aes128ctr cipher has been removed from libsodium as of `1.0.14`. We are working on updating Crypt::NaCl::Sodium now.

**Comment by Sailor-Fhem** [comment]
These are great news genio since it directly links with a problem which recently occurred while installing a module I programmed for the FHEM home automation. 
Link: https://forum.fhem.de/index.php/topic,100758.msg1079321.html#msg1079321 (Unfortunatley in German only)
Please be invited to leave a note in the forum as soon this issue is fixed.
Thanks a lot!

**Comment by tonvoon** [comment]
Hi. Is there an update to this? How can I help?
